src/components/FormLinks/FormLinks.tsx

```diff
@@ -3,23 +3,32 @@ import { Link } from '../UI'
 import './FormLinks.css'
 
 /**
- * @typedef Props
- * @props {string} classname
+ * @typedef IFormLinksProps
+ * @props {string} [classname] - the CSS classes
  */
 
-export interface Props {
+export interface IFormLinksProps {
   className?: string
 }
 
+/**
+ * Renders form component with links
+ *
+ * @param  {string} [classname] - the CSS classes
+ *
+ * @example
+ * <FormLinks className='a6y-react-auth-form' />
+ */
+
 const FormLinks = ({
   className = 'a6y-react-auth-form',
-}: Props): JSX.Element => {
+}: IFormLinksProps): JSX.Element => {
   return (
     <div className={className + '-links'}>
-      <Link className={className + '-link'} to='/sign-up'>
+      <Link className={className + '-link'} to='/forgot-password'>
```

Comment:
Let's follow the BEM standards:

* `__` - it indicates a block,
* `--` - it is a style variation.

For instance:

```
a6y-react-auth__link

a6y-react-auth__link--primary
```

Reply:
It's not crucial, so where it's easy to change, let's do this.